13_biblioteki/13_5_NAG.tex

```diff
@@ -0,0 +1,32 @@
+% !TEX encoding = UTF-8 Unicode
+\section{NAG - Numerical PVM Library}
+%%%%%%%%%%%%%%%%
+	\begin{frame}{NAG - Numerical PVM Library}
+	W oparciu o: 
```

Comment:
I'd make another itemed list instead of newllines here, for consistency and comprehensiveness ;)